### ISSUE TITLE
Add cachebusting based on HTTP headers

### ISSUE THREAD

**Issue by ksuprynowicz** [overte-org/overte#638]
Right now caching is too aggressive and things header like this could be used:
```
HTTP/1.1 200 OK
Content-Type: text/html
Content-Length: 1024
Date: Tue, 22 Feb 2022 22:22:22 GMT
Last-Modified: Tue, 22 Feb 2021 22:22:22 GMT
```
More about caching:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching#heuristic_caching

**Comment by gydence** [comment]
this PR was supposed to help with this: https://github.com/vircadia/vircadia/pull/1012

specifically, using PreferNetwork instead of PreferCache (and also, we’re not setting up the cache properly): https://github.com/vircadia/vircadia/pull/1012/files#diff-68a2781d62fbafa8031b8ce01cbf7cc0541197f7e1616805ecaa8693dedaa3c8R61

**Comment by vegaslon** [comment]
I thought that had already been merged.

**Comment by gydence** [comment]
nope, there were mixed reports of if it worked or not and I never got a chance to take another look.

**Comment by ksuprynowicz** [comment]
It looks like we already record the last modified date in the cached files so this shouldn't be too difficult
